util/vendor.py

```diff
@@ -293,7 +293,13 @@ def make_default(have_patch_dir):
                         Path('.') if have_patch_dir else None)
 
     @staticmethod
-    def apply_patch(basedir, patchfile):
+    def apply_patch(basepath, patchfile):
+        if os.path.isfile(basepath):
+            # Sometimes basepath is actually a file to which the patch should be applied.
+            # In that case, make basedir point to the containing directory instead of the file.
```

Comment:
I think this comment probably belongs just before the "if"?

Reply:
Thanks, now moved.